sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/tests/QueueListenerTests.cs

```diff
@@ -77,13 +77,13 @@ public void SetUp()
 
         public TestFixture Fixture { get; set; }
 
-        [Test]
+       [RecordedTest]
```

Comment:
no tests should be annotated with `RecordedTest` in `Microsoft.AzureWebjobs.Extensions.Storage.*` packages. We're not using recording framework at all there.